src/Storages/Kafka/KafkaBlockInputStream.cpp

```diff
@@ -119,7 +143,14 @@ Block KafkaBlockInputStream::readImpl()
                     auto columns = chunk.detachColumns();
                     for (size_t i = 0, s = columns.size(); i < s; ++i)
                     {
-                        result_columns[i]->insertRangeFrom(*columns[i], 0, columns[i]->size());
+                        if (i != error_column_index)
```

Comment:
I suppose that it may be better to handle it in separate loop. `error_column_index` always last column

Reply:
Thanks for your suggestion. I'll fix it.